Jenkinsfile.deploy

```diff
@@ -131,13 +132,13 @@ pipeline {
             }
             steps {
                 script {
-                    maven.mvnVersionsSet(getOptaplannerMavenCommand(), getProjectVersion())
+                    maven.mvnVersionsSet(getOptaplannerMavenCommand(), getProjectVersion(), !params.RELEASE)
```

Comment:
I think we should use `!isRelease()` instead of directly the params

Reply:
done